cwms-data-api/src/main/java/cwms/cda/api/project/ProjectPublishStatusUpdate.java

```diff
@@ -0,0 +1,130 @@
+    public ProjectPublishStatusUpdate(MetricRegistry metrics) {
+        this.metrics = metrics;
+    }
+
+    @OpenApi(
+            description = "Publishes a message on the office's STATUS queue that a project has "
+                    + "been updated for a specified application",
+            queryParams = {
+                @OpenApiParam(name = OFFICE, required = true, description = "The office "
+                        + "generating the message (and owning the project)."),
+                @OpenApiParam(name = PROJECT_ID, required = true, description = "The location "
+                        + "identifier of the project that has been updated"),
+                @OpenApiParam(name = APPLICATION_ID, required = true, description = "A text "
+                        + "string identifying the application for which the update applies."),
+                @OpenApiParam(name = SOURCE_ID, description = "An application-defined string "
+                        + "of the instance and/or component that generated the message. "
+                        + "If NULL or not specified, the generated message will not include "
+                        + "this item."),
+                @OpenApiParam(name = TIMESERIES_ID, description = "A time series identifier of "
+                        + "the time series associated with the update. If NULL or not "
+                        + "specified, the generated message will not include this item."),
+                @OpenApiParam(name = BEGIN, description = "The start time of the updates to "
+                        + "the time series. If NULL or not specified, the generated message "
+                        + "will not include this item."),
+                @OpenApiParam(name = END, description = "The end time of the updates to the "
+                        + "time series. If NULL or not specified, the generated message will "
+                        + "not include this item.")
+            },
+            method = HttpMethod.POST,
+            responses = {
+                @OpenApiResponse(status = STATUS_200, content = {
+                    @OpenApiContent(type = Formats.JSON, from = PublishStatusUpdateResult.class)}
+                    )},
+            tags = {TAG},
+            path = PATH
+    )
+    @Override
+    public void handle(@NotNull Context ctx) throws Exception {
+
+        try (final Timer.Context ignored = markAndTime("publish")) {
+            ProjectDao prjDao = new ProjectDao(JooqDao.getDslContext(ctx));
+            String office = requiredParam(ctx, OFFICE);
+            String appId = requiredParam(ctx, APPLICATION_ID);
+            String projectId = requiredParam(ctx, PROJECT_ID);
+
+            String sourceId = ctx.queryParam(SOURCE_ID);
+            String tsId = ctx.queryParam(TIMESERIES_ID);
+            Instant begin = queryParamAsInstant(ctx, BEGIN);
+            Instant end = queryParamAsInstant(ctx, END);
+
+            Instant timeOfMessage = prjDao.publishStatusUpdate(
```

Comment:
I don't see any reason to throw it away. We have it, its easy to return and it gives me something to observe in (minimal) test case.

Reply:
Yeah the only way to observe would be to query the message queues or subscribe to them. Seems like managing the DTO in the java server, java client, javascript client is extra maintenance for something unused. 

If we want to keep it, the DTO should be renamed to something more generic as there are other message calls that we'll end up wanting in the future (unless we start allowing clients to publish to the message broker topics directly). For example: https://bitbucket.hecdev.net/projects/CWMS/repos/hec-cwms-data-access/browse/cwms-db-dao/src/main/java/usace/cwms/db/dao/ifc/msg/CwmsDbMsg.java#96

Reply:
What do you think about PublishStatusUpdateResult being renamed to MessageTimestamp?  You asked for it to be more general so it could be used for the other Msg calls.  Or EventTimestamp?  Currently it just holds a single Instant so its not really related to Message at all.  I'd call it Instant or Timestamp but I know that would cause all sorts of confusion.  Its meant to just hold the value and as a place we can hang our annotations.  I suppose from that point of view it is related to Message b/c we're using it from Message and configuring it for Message.   So, MessageTimestamp?  MessageInstant?

Reply:
I've seen examples like: https://github.com/USACE/cwms-data-api/blob/develop/cwms-data-api/src/main/java/cwms/cda/api/EmbankmentController.java#L237 where we return a 204 with a simple status message. Can the timestamp be that message and not need a full DTO?

Reply:
or it could be a header sent back to the client `Message-Timestamp` or `"Message-Instant`

Reply:
At the meeting it was decided to remove the timestamp.